composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "bcgov-theme/bcgov-wordpress-block-theme",
     "description": "Block Theme for BC Gov",
-    "version": "1.7.0",
+    "version": "1.8.0",
```

Comment:
Im not sure a documenation addition warrants a minor version bump.
https://semver.org/